quiz.js

```diff
@@ -126,6 +126,14 @@ function showCorrectAnswer() {
   let correctAnswerLetter = questions[currentQuestion].correctAnswer;
   let correctAnswer = questions[currentQuestion].answers[correctAnswerLetter];
   setAnswerHint(correctAnswer);
+  let radioButtons = document.getElementsByName("answer");
+  [...radioButtons].map( item => {
```

Comment:
To na pewno nie było proste bo jak zauważyłaś getElementsByName zwraca NodeList a nie tablicę. Żeby zrobić map trzeba było przekonwertować. Bardzo dobrze to zrobiłaś.